### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by IamDikshaR**

### CODE REVIEW

File: src/atoms-diksha/buttons/accent/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/buttons/accent/index.astro:1]: Remove paragraph atom.

File: src/atoms-diksha/buttons/blue/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/buttons/blue/index.astro:1]: Remove paragraph atom.

File: src/atoms-diksha/buttons/grey/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/buttons/grey/index.astro:1]: Remove paragraph atom.

File: src/atoms-diksha/buttons/icon/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/buttons/icon/index.astro:1]: Remove semicolons.

File: src/atoms-diksha/paragraphs/sm/light/slate-600/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/paragraphs/sm/light/slate-600/index.astro:1]: Update: src/atoms-diksha/paragraphs/sm/light/slate/index.astro

File: src/components/common/header/desktop-header/index.astro
Comment (tanishdotdev) [inline src/components/common/header/desktop-header/index.astro:1]: Remove text-black from div in line 74.

File: src/components/common/header/index.astro
Comment (tanishdotdev) [inline src/components/common/header/index.astro:1]: Don't create a desktop header component. Add that code in this file only.

File: src/components/home/client-reviews/google-review-card/index.astro
Comment (tanishdotdev) [inline src/components/home/client-reviews/google-review-card/index.astro:1]: 1. Remove aspect-auto as size is already defined which includes both width and height in line 14.
2. Don't use h-16 w-full aspect-auto together, Use width and aspect ratio and in few cases if required then height and aspect ratio. Replace the aspect-auto with a defined aspect ratio in line 22.

File: src/components/home/client-reviews/review-card/index.astro
Comment (tanishdotdev) [inline src/components/home/client-reviews/review-card/index.astro:1]: Remove aspect-auto as size is already defined which includes both width and height in line 23 and 31.

File: src/layouts/container-client-review/index.astro
Comment (tanishdotdev) [inline src/layouts/container-client-review/index.astro:1]: Remove this.

File: src/layouts/header-layout/index.astro
Comment (tanishdotdev) [inline src/layouts/header-layout/index.astro:1]: Remove this.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

File: src/atoms-diksha/buttons/accent/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/buttons/accent/index.astro:1]: done

File: src/atoms-diksha/buttons/grey/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/buttons/grey/index.astro:1]: done

File: src/atoms-diksha/buttons/icon/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/buttons/icon/index.astro:1]: done

File: src/atoms-diksha/paragraphs/sm/light/slate-600/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/paragraphs/sm/light/slate-600/index.astro:1]: done

File: src/components/common/header/desktop-header/index.astro
Comment (IamDikshaR) [reply src/components/common/header/desktop-header/index.astro:1]: done

File: src/components/common/header/index.astro
Comment (IamDikshaR) [reply src/components/common/header/index.astro:1]: done

File: src/layouts/header-layout/index.astro
Comment (IamDikshaR) [reply src/layouts/header-layout/index.astro:1]: done

File: src/layouts/container-client-review/index.astro
Comment (IamDikshaR) [reply src/layouts/container-client-review/index.astro:1]: done

File: src/components/home/client-reviews/google-review-card/index.astro
Comment (IamDikshaR) [reply src/components/home/client-reviews/google-review-card/index.astro:1]: done

File: src/components/home/client-reviews/review-card/index.astro
Comment (IamDikshaR) [reply src/components/home/client-reviews/review-card/index.astro:1]: done

**Pull request closed without merging**